doc: text revision in pt-BR

jhserodio opened the pull request:
resolving some grammatical errors, and semantic texts for best comprehension.

jhserodio committed:
doc: text revision in pt-BR in contributing
resolving some grammatical errors, and semantic texts for best comprehension.

rust-highfive commented:
Thanks for the pull request, and welcome! The Rust team is excited to review your changes, and you should hear from @steveklabnik (or someone else) soon.

If any changes to this PR are deemed necessary, please add them as extra commits. This ensures that the reviewer can see what has changed since they last reviewed the code. Due to the way GitHub handles out-of-date commits, this should also make it reasonably obvious what issues have or haven't been addressed. Large or tricky changes may require several passes of review and changes.

Please see [the contribution instructions](https://github.com/rust-lang/rust/blob/master/CONTRIBUTING.md) for more information.

jhserodio commented:
Thanks @rust-highfive , i intend to submit others pull requests, with more pt_BR language revisions.

jhserodio committed:
fix: community page pt_BR

resolve little problems of semantic

jhserodio commented:
i will review all the site in pt_BR :)

jhserodio committed:
fix: conduct page gramatical errors

resolve errors in gramaticals, and the use of dot, semicolum and comma. resolve the wrong link in community

jhserodio committed:
fix: resolve gramatical errors in documentation.md

resulve gramaticals and semanticals errors in documantion page

jhserodio committed:
fix: revision contribute-bugs

only revision in translates terms

jhserodio committed:
fix: gramatical errors in contribute-community

resolve some gramaticals errors

jhserodio committed:
fix: errors in contribute-compiler

resolve grammatical and semantic errors

jhserodio committed:
fix: gramatical errors in contribute-documentation

resolve some problems with gramatic and translateds terms

jhserodio committed:
fix: gramaticals errrors in contribute-libs

some gramaticals errors

jhserodio committed:
fix: gramatical errors in contribute-tools

some gramaticals errors and  terms resolves

leoyvens commented on pt-BR/conduct.md:
```diff
@@ -9,12 +9,12 @@ title: O Código de Conduta de Rust &middot; A linguagem de programação Rust
 **Contato**: [rust-mods@rust-lang.org](mailto:rust-mods@rust-lang.org)
 
-* Nós estamos comprometidos com fornecer um ambiente amigável, seguro e acolhedor para todos, independente di nível de experiência,
-  sexo, identidade e expressão de gênero, orientação sexual, falta de habilidade, aparência pessoal, tamanho do corpo, raça, etinia,
```
This "e" was intentional

jhserodio replied:
sorry, but I thought 'e' was a grammatical error, I also found the same about 'di'

leoyvens commented on pt-BR/conduct.md:
```diff
@@ -9,12 +9,12 @@ title: O Código de Conduta de Rust &middot; A linguagem de programação Rust
+* No IRC, por favor evite usar nomes sugestivos ou outros nomes que podem denegrir a imagem de um ambiente amigável, seguro e acolhedor para todos.
 * Por favor, seja gentil e cortês. Não há necessidade de ser mau ou rude.
-* Respeite as pessoas que diferem de sua opinião e que escolhas de design ou implementação levem à problemas e inúmeros custos.   Raramente há apenas uma única escolha certa.
+* Respeite as pessoas que diferem de sua opinião e que escolhas de design ou implementação que levem à problemas e inúmeros custos. Raramente há apenas uma única escolha certa.
```
This "que" doesn't make much sense here.

jhserodio replied:
I agree with you, but this is for the whole paragraph: "Respeite as pessoas que diferem de sua opinião, escolhas de design ou implementação que levem à problemas e inúmeros custos. Raramente há apenas uma única escolha certa.". This is better?

leoyvens replied:
better

leoyvens commented on pt-BR/conduct.md:
```diff
@@ -30,21 +30,21 @@ title: O Código de Conduta de Rust &middot; A linguagem de programação Rust
 5. Se o usuário retornar e continuar a causar problemas, ele será banido, ou seja, definitivamente excluído.
 6. Moderadores podem escolher des-banir um usuário se foi uma primeira ofensa e eles oferecem uma desculpa sincera e genuina.
-7. Se um moderador bane alguem e você acha que foi injusto, por favor converse com esse moderador, ou com um moderador diferente, **em privado**. Reclamações sobre banimentos no canal público não são permitidos.
+7. Se um moderador baneficiar alguem e você achar que foi injusto, por favor converse com esse moderador, ou com um moderador diferente, **em privado**. Reclamações sobre banimentos no canal público não são permitidos.
```
"baneficiar" -> "banir"

jhserodio replied:
sorry, my fault.

leoyvens commented on pt-BR/contribute-bugs.md:
```diff
@@ -7,7 +7,7 @@ title: Contribuindo para Rust &mdash; encontrando, fazendo triagem e solucionand
 A manutenção diária do projeto gira em torno do [seguidor de problemas][issue tracker]
 e das [pull requests][PR] de Rust, e mais ajuda é sempre necessária. O jeito mais
-básico de se começar a contribuir para Rust é olhar para rótulos [E-easy] e [E-mentor].
+básico de se começar a contribuir para Rust é olhar para *labels* [E-easy] e [E-mentor].
```
Why untranslate "labels" everywhere?

jhserodio replied:
Labels are popular terms for Brazilian developers, say labels at work, personal projects, we use terms in English that often represent a thing. It is more natural;

leoyvens replied:
maybe, but when in doubt I prefer to translate too much instead of too little.

jhserodio replied:
i agree.

leoyvens commented on pt-BR/contribute-compiler.md:
```diff
@@ -36,12 +36,12 @@ são muitas vezes independentes.
+performance do compilador, e o código Rust pode ser medido com profilers como `perf`, no Linux.
 
-As maiores novas features em Rust passam por um processo de [Pedido por Comentários (RFC)][rfc],
+As Pirncipais novas features em Rust passam por um processo de [Pedido por Comentários (RFC)][rfc],
```
"Pirncipais" -> "principais"

jhserodio replied:
i agreee, my fault.

leoyvens commented on pt-BR/documentation.md:
```diff
@@ -7,9 +7,9 @@ title:  Documentação Rust &middot; A linguagem de programação Rust
-te dar uma boa ideia de como Rust se parece, te mostra como instalá-lo, e explica
-a sintaxe e os conceitos. Ao completar o livro, você será um desenvolvedor intermediário
-de Rust, e vai ter uma bom fundamento das ideias por trás de Rust.
+te dar uma boa visao sobre Rust, te mostrar como instalá-lo, explicar
```
visao -> visão

jhserodio replied:
thanks.

leoyvens reviewed (changes requested):
@jhserodio Thanks for helping out with the website! We also have a translation for the book going in https://github.com/rust-br/rust-book-pt-br/blob/master/CONTRIBUTING-pt-br.md, would you like to help with that? If you want to, come to the Rust Brasil telegram to get in touch with the community https://t.me/rustlangbr

Commit:
fix: text revision in pt-BR

jhserodio committed:
Merge branch 'master' into master
